js/test/Ice/binding/Client.ts

```diff
@@ -7,9 +7,9 @@ import { TestHelper } from "../../Common/TestHelper.js";
 const ArrayUtil = Ice.ArrayUtil;
 const test = TestHelper.test;
 
-const isConnectionFailed = (ex: Ice.Exception) =>
-    (!TestHelper.isBrowser() && ex instanceof Ice.ConnectionRefusedException) ||
-    (TestHelper.isBrowser() && ex instanceof Ice.ConnectFailedException) ||
+const isConnectionFailed = (defaultProtocol: string, ex: Ice.Exception) =>
```

Comment:
The exception is different base on the protocol, previously using isBrowser == false was equivalent to TCP transport, but that is no longer the case.